BWWalkthroughExampleTests/BWWalkthroughExampleTests.swift

```diff
@@ -1,36 +0,0 @@
-//
```

Comment:
It should just not be referenced in our source code, it should not be removed from the repository.